lib/command_line_reporter/row.rb

```diff
@@ -58,12 +58,15 @@ def output
           # | xxxxxxxxxxx |            | (4)
           # +-------------+------------+
           if col.screen_rows[sr].nil?
-            line << ' ' * col.width
+            line << ' ' * col.width << ' '
           else
-            line <<  self.columns[mc].screen_rows[sr]
+            line << self.columns[mc].screen_rows[sr] << ' '
           end
 
-          line << ' ' + ((self.border) ? "#{border_char} " : '')
+          if self.border
+            line << border_char
+            line << ' ' if mc < self.columns.size - 1
```

Comment:
Re: redundant `self` - noticed that while making the changes but the rest of the code also has the redundant usage so I kept it consistent.